pubtools/_quay/container_image_pusher.py

```diff
@@ -266,14 +270,18 @@ def push_container_images(self):
                 .get("image", {})
                 .get("sources_for_nvr", None)
             )
-            if not sources_for_nvr and not source_ml:
+            if (
+                not sources_for_nvr
+                and not source_ml
+                and not self.target_settings.get("allow_v1_containers", True)
```

Comment:
It's not version 1, is it? It's version 2 schema 1

Reply:
it's reported as application/vnd.docker.distribution.manifest.v1+json. But we call it v2sch1 probably because there was api with v1/ and now it's v2/ and therefore we call it v2 sch1, but docker tracks it as v1.